Fixed that the `reason` in the `authn_complete_event` event was not displayed correctly.
